### PR TITLE
Add Ambient.Setoid.Trivial (Squash)

### PR DESCRIPTION
**Pull request by jonsterling**

### CODE REVIEW

File: src/Ambient/Setoid/Trivial.agda [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+module Ambient.Setoid.Trivial where
+
+open import Agda.Primitive
+import Ambient.Setoid.Base as S
+open import Type as T
+  using (_,_)
+
+s : ∀ {d} ..{ℓᵒ ℓʰ}
+  → (A : T.t ℓᵒ)
+  → S.t d ℓᵒ ℓʰ
```
Comment (jonsterling) [inline src/Ambient/Setoid/Trivial.agda]: I wasn't sure whether I should have parameterized over `ℓʰ` or not

**Comment by freebroccolo** [comment]
Cool, thanks. I think for the terminal setoid I didn't make the parameter explicit but I don't have a strong opinion about it. You can always just make a wrapper if it's convenient.

**Comment by freebroccolo** [comment]
Oh, btw, I think I have this defined also at https://github.com/freebroccolo/agda-groupoids/blob/master/src/Ambient/Setoid/Base.agda#L28-L42 . Maybe it would be better to rename it?

**Comment by freebroccolo** [comment]
There is a similar one for setoids to groupoids also I think.

**Comment by jonsterling** [comment]
Oh, I didn't see that one, sorry! I'll submit another PR which deletes the new one and renames the other one.

**Comment by freebroccolo** [comment]
No worries… this stuff is kind of a maze :)

**Comment by jonsterling** [comment]
You know, I think maybe I should just delete the old one and keep the new one, since it lives in its own module; the reason is, once I add quotients & subsets, we can just define squash in terms of those quite easily.

**Comment by jonsterling** [comment]
Eh, too avoid too much churn (having to deal with moving stuff around in the Groupoid module), I'm just going to delete the new one and rename the old one to be more clear.

**Comment by freebroccolo** [comment]
Okay, well either way is fine with me.

**Comment by freebroccolo** [comment]
Just a note: in the refactored version I've put this under `Construction.Chaotic` since it apparently corresponds to what is sometimes called the chaotic or [indiscrete groupoid](https://ncatlab.org/nlab/show/indiscrete+category). I've also added the discrete and free setoids.

**Pull request merged**